packages/nextjs/components/MotionElements.tsx

```diff
@@ -7,5 +7,6 @@ const MotionMain = motion.main;
 const MotionH1 = motion.h1;
 const MotionP = motion.p;
 const MotionA = motion.a;
+const MotionS = motion.section;
```

Comment:
Maybe we can name it `MotionSection`, so it follows the established naming pattern.